libtransmission/peer-io.cc

```diff
@@ -439,12 +444,12 @@ static void utp_on_write(void* vio, unsigned char* buf, size_t buflen)
     TR_ASSERT(tr_isPeerIo(io));
 
     int rc = evbuffer_remove(io->outbuf, buf, buflen);
-    tr_logAddTraceIo(io, "utp_on_write sending %zu bytes... evbuffer_remove returned %d", buflen, rc);
+    tr_logAddTraceIo(io, fmt::format("utp_on_write sending {} bytes... evbuffer_remove returned {}", buflen, rc));
     TR_ASSERT(rc == (int)buflen); /* if this fails, we've corrupted our bookkeeping somewhere */
 
     if (rc < (long)buflen)
     {
-        tr_logAddWarnIo(io, "Short write: %d < %ld", rc, (long)buflen);
+        tr_logAddWarnIo(io, fmt::format(_("Short write: {} < {}"), rc, buflen));
```

Comment:
Named placeholders. Maybe a better description? (Not at all sure translators/users will understand what it's about)

Reply:
517b6c320a6bce6cca381f6f54ea38d01134256a